.github/actions/pricing-list-action/DockerFile

```diff
@@ -0,0 +1,15 @@
+FROM golang:1.21
+
+WORKDIR /go/src/app
+
+COPY . .
+
+RUN go mod download
+
+RUN go get -u github.com/gocarina/gocsv
+RUN go get -u github.com/tidwall/gjson
+
+COPY entrypoint.sh /entrypoint.sh
+RUN chmod +x /entrypoint.sh
+
+ENTRYPOINT ["/entrypoint.sh"]
```

Comment:
Make it into go file and run the command below, you already define the go container in line 1

CMD ["go run", "/go/src/app/main.go"]